cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/upgrade/operation/AwsRdsVersionOperations.java

```diff
@@ -64,12 +64,12 @@ private String getPostgresFamilyVersion(String engineVersion) {
             if (engineVersionMatcher.matches()) {
                 String engineMajorVersion = engineVersionMatcher.group(GROUP_MAJOR_VERSION);
                 int engineMajorVersionNumber = Integer.parseInt(engineMajorVersion);
-                if (VERSION_FAMILY_9.getMajorVersionFamily() <= engineMajorVersionNumber && VERSION_13.getMajorVersionFamily() >= engineMajorVersionNumber) {
+                if (VERSION_FAMILY_9.getMajorVersionFamily() <= engineMajorVersionNumber && VERSION_14.getMajorVersionFamily() >= engineMajorVersionNumber) {
```

Comment:
are we still able to start DB with postgres 9?

Reply:
good question, I didn't want to touch that as I am not aware of the potential consequences